store/generated_key/v2/eigenda.go

```diff
@@ -31,14 +42,14 @@ var _ common.GeneratedKeyStore = (*Store)(nil)
 
 func NewStore(
 	log logging.Logger,
-	putRetries uint,
+	putRetries int,
 	disperser *payloaddispersal.PayloadDisperser,
 	retriever clients.PayloadRetriever,
 	verifier clients.ICertVerifier,
 ) (*Store, error) {
 	return &Store{
 		log:        log,
-		putRetries: putRetries,
+		putRetries: utils.MapPutRetries(putRetries),
```

Comment:
Feel like MapPutRetries should be only called at the retry-go call site. Our configs should be self-consistent and only contain our own meaning of retry imo